src/main/java/net/snowflake/client/log/StdErrOutThresholdAwareConsoleHandler.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2012-2019 Snowflake Computing Inc. All rights reserved.
+ */
+package net.snowflake.client.log;
+
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+import java.util.logging.SimpleFormatter;
+import java.util.logging.StreamHandler;
+
+class StdErrOutThresholdAwareConsoleHandler extends StreamHandler {
```

Comment:
Should it be marked as internal?

Reply:
does not have to, the class is not public